skills/skill-council/SKILL.md

```
@@ -15,6 +15,18 @@ Use this skill for `/octo:council` and council-style requests.
 "${CLAUDE_PLUGIN_ROOT:-$HOME/.claude-octopus/plugin}/scripts/orchestrate.sh" council <user arguments>
 ```
 
+**Sandbox-safe invocation (required under Claude Code's sandbox).** `orchestrate.sh`
+must be the FIRST token of the command. The sandbox's unsandboxed-command allowlist is a
+leading-anchored prefix match, so a command that begins with `cd …`, an inline
+`VAR=… ` env-assignment, or any other prefix will NOT match an `…/orchestrate.sh *`
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)